primer3/primerdesign.pxd

```diff
@@ -0,0 +1,546 @@
+'''
+pd_helpers.pxi
```

Comment:
I suppose to be consistent we should include the license notice in `pxd/pxi` files as well (btw the file extension is `pxd` but this comment lists `pxi`)

Reply:
missed this update

Reply:
fixed